client/settings/general-settings-section/upe-opt-in-banner.js

```diff
@@ -0,0 +1,43 @@
+	/>
+);
```

Comment:
This is the same image that was in `client/payment-method-icons/all-payment-methods/icon.svg` - with a little less markup